app/views/provider_interface/reports/hesa_exports/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_for :title, t('page_titles.provider.export_hesa_data') %>
+
+<%= breadcrumbs({
```

Comment:
This should be in a `content_for :before_content`, and the same for the other reports export page